src/types/PointDetailsType.ts

```diff
@@ -0,0 +1,28 @@
+// Response returned by azimuth#getPoint
+export type PointDetails = {
+  0: string;
+  1: string;
+  2: string;
+  3: string;
+  4: string;
+  5: string;
+  6: string;
+  7: string;
+  8: string;
+  9: string;
```

Comment:
I think all of these have named equivalents below, and so we don't care about the numbered accessors? Might be fine to just leave these out, they're not really helpful.